prov/psm/configure.m4

```diff
@@ -46,7 +46,6 @@ AC_DEFUN([FI_PSM_CONFIGURE],[
 	psm_CPPFLAGS="$CPPFLAGS $psm_CPPFLAGS"
 	psm_LDFLAGS="$LDFLAGS $psm_LDFLAGS"
 	psm_LIBS="$LIBS $psm_LIBS"
```

Comment:
I understand why you put `$CPPFLAGS` and `$LDFLAGS` and `$LIBS` in the `psm_` versions (because you're making a new convenience LT library and you didn't want to list both sets of flags in the top-level Makefile.am).

But it still seems a little semantically weird to me -- i.e., `psm_CPPFLAGS` is now the main set of `CPPFLAGS` *and* the PSM-specific `CPPFLAGS`.  My $0.02 is to keep them separate: let the PSM-specific flags be PSM-specific flags, and up in the top-level Makefile.am, list them both -- e.g.:

```Makefile
libpsmx_la_LDFLAGS = $(LDFLAGS) $(psm_LDFLAGS)
```

I realize this is a stylistic opinion, so feel free to ignore me.  But I thought I'd at least mention it...